libs/ledger-live-common/src/socket/index.ts

```diff
@@ -222,27 +243,62 @@ export function createDeviceSocket(
           }
 
           case "success": {
-            // a final success event with some data payload
+            // A final success event with some data payload
             const payload = input.result || input.data;
-            if (payload)
+
+            tracer.trace("Socket in: success", {
+              payload,
+              inBulkMode,
+              nonce: input?.nonce,
+              uuid: input?.uuid,
+              session: input?.session,
+            });
+
+            // Once entered in bulk mode, we close the websocket and don't react to any other messages
+            if (inBulkMode) break;
```

Comment:
this is just enforcing the existing logic of closing the web socket / not listening once we're in bulk mode, LGTM 👍

Reply:
good catch too!

Reply:
yes :)